pkg/transferer/load.go

```diff
@@ -109,6 +113,13 @@ func (c *LoadChecker) updateStatus() {
 		status = LoadStatusWait
 		log.Warn().Err(err).Msgf("Error while checking metrics load")
 	}
+	if status == LoadStatusWait {
+		c.waitAttemptsCounter++
```

Comment:
we want to limit sequential waits so the program doesn't hang if the load is not decreasing at all

in a case, when load status changes every few iterations (wait[5] -> ok[2] -> wait[5] ...) we want to proceed normally

Reply:
Resolved in d2bdbaca363c55fe9de7934d01ac8667233954f2